### PR TITLE
feat(chain-adapters): return token balances from cosmossdk getaccount method

### PR DESCRIPTION
**Pull request by pastaghost**
* CosmosSdkBaseAdapter::getAccount() returns asset balance data
* this works for both cosmos and osmosis accounts and return data has been verified with the chain-adapter CLI.

### CODE REVIEW

File: packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts [hunk 1]
```diff
@@ -191,7 +192,12 @@ export abstract class CosmosSdkBaseAdapter<T extends CosmosSdkChainId> implement
           })),
         }))
 
-        return { ...data, delegations, redelegations, undelegations, rewards }
+        const assets = data.assets.map<CosmosSDKToken>((asset) => ({
+          amount: asset.amount,
+          denom: asset.denom,
```
Comment (gomesalexandre) [inline packages/chain-adapters/src/cosmossdk/CosmosSdkBaseAdapter.ts:197]: Can we redeclare `generateAssetIdFromOsmosisDenom` in lib and make this `assetId` for parity with `EVMBaseAdapter`? Lib shouldn't speak token addresses / denoms but rather all CAIP

**Review by gomesalexandre — COMMENTED** [review]
Looks good to me, added one comment WRT exposing the raw denom vs. the massaged `AssetId`. Web should now have all it needs to understand osmosis CAIPs, so we should expose the account tokens as such

File: packages/chain-adapters/src/cosmossdk/types.ts [hunk 2]
```diff
@@ -89,7 +90,7 @@ export type FeeData = {
 }
 
 export type CosmosSDKToken = {
-  denom: string
+  assetId: AssetId | string
```
Comment (gomesalexandre) [inline packages/chain-adapters/src/cosmossdk/types.ts:93]:
```suggestion
  assetId: AssetId
```

**Review by gomesalexandre — COMMENTED** [review]
Comment

**Review by 0xdef1cafe — APPROVED** [review]

**Pull request merged**